### PR TITLE
The order of the parameters of zkSessionTimeOutMs and zkConnectionTimeoutMs for creating the ZkUtils object is reversed.

### PR DESCRIPTION
**Pull request by wangzzu**

### CODE REVIEW

**Comment by wangzzu** [comment]
the SimpleAclAuthorizer class has an error in using ZkUtils

File: core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala [hunk 1]
```diff
@@ -107,8 +107,8 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     val zkSessionTimeOutMs = configs.get(SimpleAclAuthorizer.ZkSessionTimeOutProp).map(_.toString.toInt).getOrElse(kafkaConfig.zkSessionTimeoutMs)
 
     zkUtils = ZkUtils(zkUrl,
-                      zkConnectionTimeoutMs,
                       zkSessionTimeOutMs,
+                      zkConnectionTimeoutMs,
```
Comment (ijuma) [inline core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala]: I suggest using named arguments to avoid this kind of bug in the future.

File: core/src/main/scala/kafka/server/KafkaApis.scala [hunk 2]
```diff
@@ -926,6 +926,7 @@ class KafkaApis(val requestChannel: RequestChannel,
       val responseBody = new GroupCoordinatorResponse(Errors.GROUP_AUTHORIZATION_FAILED.code, Node.noNode)
       requestChannel.sendResponse(new RequestChannel.Response(request, new ResponseSend(request.connectionId, responseHeader, responseBody)))
     } else {
+      // the partition of __consumer_offsets which store the offset of this group
```
Comment (ijuma) [inline core/src/main/scala/kafka/server/KafkaApis.scala]: This seems unrelated?

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, left a couple of comments.

File: core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala (hunk 1 above)
Comment (wangzzu) [reply core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala]: Ye, the bug likes this is not easy to find.

**Comment by guozhangwang** [comment]
LGTM besides @ijuma's suggestion.

**Comment by wangzzu** [comment]
Thanks for @ijuma 's suggestion, I have updated the code.

**Comment by ijuma** [comment]
Thanks, LGTM. I also tweaked the creation of `ZkUtils` in `KafkaServer` to use named parameters for consistency. Merged to trunk.
